.github/workflows/release.yaml

```diff
@@ -0,0 +1,50 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v1
+      - name: build binaries
+        env:
+          CROSS: 1
+          VERSION: ${{ github.ref_name }}
+        run: |
+          make build-rancher
+
+      - name: package
+        run: |
+          make package-rancher
+
+            - name: retrieve GPG Credentials
+              uses: rancher-eio/read-vault-secrets@main
+              with:
+                secrets: |
+                  secret/data/github/repo/${{ github.repository }}/key/app-credentials passphrase | GPG_PASSPHRASE ;
+                  secret/data/github/repo/${{ github.repository }}/key/app-credentials privateKey | GPG_KEY
```

Comment:
Can you fix the indents?

Reply:
Good catch, copy paste error on my part, should be fixed